### PR TITLE
Don't capitalize particular words in breadcrumbs. closes #1328

### LINKED ISSUE #1328: breadcrumb fix for Accredited Rep (Get Help Filing a Claim) page
On this page:
https://staging.vets.gov/disability-benefits/apply-for-benefits/help/index.html

In the breadcrumbs, the word "For" in "Apply For Benefits" needs to be lowercased.

### PR DESCRIPTION
**Pull request by webinista**
Adds list of words, and a check for whether words in an array should
be capitalized.

modified:   _plugins/breadcrumbs.rb

### CODE REVIEW

**Comment by webinista** [comment]
@emilyville i'm assigning it to you for triage, but it's a Ruby change, so reassign as needed.

**Comment by ErieMeyer** [comment]
konmari

**Comment by emilyville** [comment]
@plusjeff I am sending your way!!

File: _plugins/breadcrumbs.rb [hunk 1]
```diff
@@ -1,9 +1,17 @@
 module Jekyll
   class BreadcrumbsTag < Liquid::Tag
-
-    def humanize(s)
+  
+	def humanize(s)
```
Comment (akainic) [inline _plugins/breadcrumbs.rb]: The indentation is a little weird here, as well as for the if block below

**Comment by akainic** [comment]
LGTM with a small nit!

**Comment by emilyville** [comment]
:+1:

**Pull request merged**